internal/clioptions/clioptions.go

```diff
@@ -49,6 +49,9 @@ type CLIOptions struct {
 
 	ServiceAccountRole string
 	OutputPath         string
+
+	MarketplaceResourceFilePaths []string
+	MarketplaceResourcesDirPath  string
```

Comment:
Why do we have Marketplace specific options inside clioptions?

Reply:
Because the same `clioptions` is passed to all commands, and all flags are defined here regardless of being exclusive to a specific command or not. @JGiola confirmed me this is the chosen way to add flags to commands